app/repository/account.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy.orm import Session
```

Comment:
Here we can first see another layer concern separation that points dependencies just inwards leading us to good design patterns practices.
Another improvement could be using CQRS to even separate simple queries in a different connection and from business layer.